### ISSUE TITLE
Iconify function does not work properly (again)

### ISSUE THREAD

**Issue by qirlib** [glfw/glfw#538]
Hello everyone,
A few days later after previous bug (glfwIconifyWindow incorrect behaviour, see https://github.com/glfw/glfw/issues/531) was closed I tried to use glfwIconifyWindow and noticed that video mode restoration works. But then I see my desktop with a black window that is supposed to represent my application, part of my background image and no other windows on this monitor. When I interact with Cinnamon UI or just click using right mouse button normal windows appear and everything works properly.

P. S. Linux Mint 17.1 Cinnamon 64-bit, nVidia driver version 352.21, X11 (two monitors as one virtual screen)

**Comment by elmindreda** [comment]
Can you reproduce this with `iconify -f`?

**Comment by qirlib** [comment]
Yes. Seems like iconification event really happens, but screen is not refreshed. I can see that my cursor image changes when it moves over other windows' decorations, although they're invisible to me because screen image has not been refreshed. Seems when I interact with other windows it triggers something that cause screen refreshment event to happen.

**Comment by elmindreda** [comment]
Sounds like a compositor bug. Unable to reproduce on Ubuntu 14.10 with Cinnamon.

**Comment by elmindreda** [comment]
Still unable to reproduce this.

**Comment by qirlib** [comment]
Tested again. Was unable to reproduce on Debian 8 (NVidia drivers), Ubuntu 15.10 (nouveau), nearly fresh-installed Mint+Cinnamon system (with or without proprietary NVidia drivers enabled). It seems that the glitch what I saw was either the result of some possible misconfiguration in my old OS installation (which is lost now - I formatted the hard drive) or external bug that they fixed long time ago.
As for me I think it's safe to close this issue.
